source/catalog.html

```diff
@@ -215,112 +224,115 @@ <h1 class="main-title">Каталог продукции</h1>
             </tr>
           </table>
 
-          <button class="primary-button product-card__button" type="button">Заказать</button>
+          <a href="#" class="primary-button product-card__button">Заказать</a>
         </li>
 
         <li class="product-card">
-          <span>Показать еще 100500 товаров</span>
-          <span>На самом деле вкусов гораздо больше!</span>
+          <strong>Показать еще 100500 товаров</strong>
+          <p>На самом деле вкусов гораздо больше!</p>
           <button class="product-card__button" type="button">Показать все</button>
         </li>
       </ul>
 
-      <div class="extra-product-catalog">
-        <h2 class="secondary-title-catalog">Дополнительные товары</h2>
+      <section class="extra-product-catalog">
+        <h2 class="secondary-title-catalog extra-product-catalog__title">Дополнительные товары</h2>
         <ul>
           <li class="extra-product-card">
-            <span class="extra-product-card__title">Сахарозаменитель</span>
-            <table class="extra-product-card__table">
-              <tr class="extra-product-card__table-row">
-                <td class="extra-product-card__table-data">1 упаковка (100 г)</td>
-                <td class="extra-product-card__table-data">200 &#8381;</td>
-              </tr>
-            </table>
-            <button class="primary-button extra-product-card__button" type="button">Заказать</button>
+            <h3 class="extra-product-card__title">Сахарозаменитель</h3>
+              <dl class="extra-product-card__table">
+                <dt class="visually-hidden">Объем</dt>
+                <dd class="extra-product-card__data">1 упаковка (100 г)</dd>
+
+                <dt class="visually-hidden">Цена</dt>
+                <dd class="extra-product-card__data">200 &#8381;</dd>
+              </dl>
+
+            <a href="#" class="primary-button extra-product-card__button">Заказать</a>
```

Comment:
С вложенностью немного путаница вышла
```suggestion
            <h3 class="extra-product-card__title">Сахарозаменитель</h3>
            <dl class="extra-product-card__table">
              <dt class="visually-hidden">Объем</dt>
              <dd class="extra-product-card__data">1 упаковка (100 г)</dd>

              <dt class="visually-hidden">Цена</dt>
              <dd class="extra-product-card__data">200 &#8381;</dd>
            </dl>

            <a href="#" class="primary-button extra-product-card__button">Заказать</a>
```